source/includes/elastic-agents/_031-list-agents.md

```diff
@@ -1,55 +1,56 @@
 ## List Agents
 
 ```java
-import com.thoughtworks.go.plugin.api.*;
-import com.thoughtworks.go.plugin.api.annotation.Extension;
-import com.thoughtworks.go.plugin.api.logging.Logger;
-import com.thoughtworks.go.plugin.api.request.*;
-import com.thoughtworks.go.plugin.api.response.*;
-import com.google.gson.Gson;
-import java.util.*;
-
-@Extension
-public class DockerPlugin implements GoPlugin {
```

Comment:
The intention here was to show the minimal code in a `GoPlugin` that can handle and respond to each request.